src/Makefile

```diff
@@ -79,6 +79,16 @@ $(OUTPUT)/retsnoop.skel.h: $(OUTPUT)/mass_attach.bpf.o
 $(OUTPUT)/retsnoop.o: $(OUTPUT)/retsnoop.skel.h
 $(OUTPUT)/mass_attacher.o: $(OUTPUT)/retsnoop.skel.h $(OUTPUT)/calib_feat.skel.h
 
+../sidecar/target/release/addr2line::
+	$(call msg,CARGO,addr2line)
+	$(Q)cd ../sidecar; cargo build --release
```

Comment:
I learned (https://github.com/rust-lang/cargo/issues/1485#issuecomment-89804438) that cd can be avoided, let's try that here?